_sources/scripts/run-apache.sh

```diff
@@ -68,11 +68,19 @@ make_dir_writable "$MW_VOLUME" -not '(' -path "$MW_VOLUME/images" -prune ')'
 
 # Check and update permissions of wiki images in background.
 # It can take a long time and should not block Apache from starting.
-/update-images-permissions.sh &
+if isTrue "$FAST_BOOT"; then
```

Comment:
So overall this makes sense, but I suggest inverting the logic, since the default is that we want to boot fast - and also the name could be a bit clearer. Suggest something like
```
if isTrue "$SYNCHRONOUS_SCRIPTS"; then
    /update-images-permissions.sh
else
    /update-images-permissions.sh &
fi
```
and then having the default of SYNCHRONOUS_SCRIPTS be 0 instead of 1.

We should also update the comments to say that *normally* we shouldn't block Apache, but it can be configured to do so

Reply:
good points, thanks